### PR TITLE
AzureCP PR: API Version Update

### PR DESCRIPTION
**Pull request by huangpf**

### CODE REVIEW

File: arm-compute/compositeComputeClient.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "description": "Composite Swagger for Compute Client"
     },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-06-15/swagger/compute.json",
+    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
```
Comment (huangpf) [inline arm-compute/compositeComputeClient.json]: @amarzavery 
Can I just put ```2016-03-30/swagger/compute.json``` here?

File: arm-compute/compositeComputeClient.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
     "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-11-01-preview/swagger/containerService.json"
```
Comment (huangpf) [inline arm-compute/compositeComputeClient.json]: similar.

File: arm-compute/compositeComputeClient.json (hunk 1 above)
Comment (amarzavery) [inline arm-compute/compositeComputeClient.json]: nope. This is not supported for now. AutoRest does not know that "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/" is the base url and you are referring another document relative to the base.
Anyways, using the full url is easy for now. We will think of adding basePath and then have documents relative to the basepath. But this is of low priority in the current scheme of things.

File: arm-compute/compositeComputeClient.json (hunk 2 above)
Comment (amarzavery) [inline arm-compute/compositeComputeClient.json]: Please don't replace the old swagger spec with the new version. We need it for getting history of what changed over time. Add a new folder for the api-version and a new swagger spec over there. Take a look at this: https://github.com/Azure/azure-rest-api-specs/tree/master/arm-storage

Comment (huangpf) [inline arm-compute/compositeComputeClient.json]: ok

File: arm-compute/compositeComputeClient.json (hunk 1 above)
Comment (huangpf) [inline arm-compute/compositeComputeClient.json]: It's a deduction that AR shall be able to figure out -- the location of other relative specs shall be based on the composite file.

**Comment by huangpf** [comment]
retest this please

**Comment by huangpf** [comment]
@amarzavery 
Why tests start to fail now? 
```
1 failing
  1) Azure Swagger Schema Validation "before all" hook:
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
```

**Comment by devigned** [comment]
sounds like a github or network related issue. Seems like it's trying to pull down schema and failing to do so in the 2 second max.

**Comment by huangpf** [comment]
Thanks @devigned. Reopening it solves the problem~

**Comment by amarzavery** [comment]
Then should this be merged ?

**Comment by huangpf** [comment]
@amarzavery Yes, please. Thank you!

**Pull request merged**